src/diffusers/models/modeling_utils.py

```diff
@@ -292,6 +295,149 @@ def disable_xformers_memory_efficient_attention(self):
         """
         self.set_use_memory_efficient_attention_xformers(False)
 
+    def add_adapter(self, adapter_config, adapter_name: Optional[str] = None) -> None:
+        r"""
+        If you are not familiar with adapters and PEFT methods, we invite you to read more about them on the PEFT
```

Comment:
I would suggest to make the first sentence of the docstring a short description of what it does, and only then add this sentence about PEFT.

Reply:
Makes sense, this should be done now!